packages/ProgressIndicators/src/index.tsx

```diff
@@ -1 +1,2 @@
-export * from './ProgressBar';
+import ProgressBar from './ProgressBar';
+export { ProgressBar };
```

Comment:
What would happen if a user tries importing the package like so?
```ts
import ProgressBar from '@onaio/progress-indicators'; 🤔 
```
What _should_ happen?